### PR TITLE
Dataset/LSA-T

### LINKED ISSUE #32: Add LSA-T: "The first continuous Argentinian Sign Language dataset for Sign Language Translation"
> Sign language translation (SLT) is an active field of study that encompasses human-computer interaction, computer vision, natural language processing and machine learning. Progress on this field could lead to higher levels of integration of deaf people. This paper presents, to the best of our knowledge, the first continuous Argentinian Sign Language (LSA) dataset. It contains 14,880 sentence level videos of LSA extracted from the CN Sordos YouTube channel with labels and keypoints annotations for each signer. We also present a method for inferring the active signer, a detailed analysis of the characteristics of the dataset, a visualization tool to explore the dataset and a neural SLT model to serve as baseline for future experiments. 

Info:
* Languages: Spanish, Argentinian Sign Language
* Paper: https://arxiv.org/abs/2211.15481
* Dataset URL: https://github.com/midusi/LSA-T
* Samples: 22 hours of sentences.
* Features: video:RGB, text:es, pose:AlphaPose 
* Signers:103
* Signs: Not sure. They discuss the Spanish vocabulary, but not the LSA exactly.

They also made their own dataloader in PyTorch. 

Poses calculated with AlphaPose: https://github.com/MVIG-SJTU/AlphaPose, "with the Halpe full-body keypoints format" from https://github.com/Fang-Haoshu/Halpe-FullBody, introduced in AlphaPose. Helpfully they also release their models and code at https://github.com/midusi/keypoint-models

### PR DESCRIPTION
**Pull request by cleong110**
Additional Checklist for datasets:

When adding a dataset, follow the following steps. [This pull request](https://github.com/sign-language-processing/sign-language-processing.github.io/pull/28) provides an example:

- [x] Fork the repo
- [x] sync forks
- [x] git checkout master
- [x] git pull
- [x] New branch: dataset/something
- [x] Create a JSON along the lines of the schema below. e.g. FOO.json
- [x] Add the JSON to `src/datasets`. e.g. `src/datasets/FOO.json`
- [x] Add BibTex to `src/references.bib`.
  - [x] prepend the citation key with `dataset`. e.g. `dataset:sehyr2021asl`
  - [x] "language" field should not need "sign language". No need to say "American Sign Language", "American" will do.
  - [x] Very concise "samples" field, the table does not have a lot of space to display it.
- [x] Commit/push the changes
- [ ] Make a pull request!

Schema:

```json
{
  "pub": {
    "name": string, # this gets used as the name of the dataset, e.g. "WLASL"
    "year": integer or null,
    "publication":string or null, # this matches a key in references.bib, e.g. "dataset:joshiISLTranslateDatasetTranslating2023"
    "url": string or null # URL to access it. e.g. "https://www.sign-lang.uni-hamburg.de/dgs-korpus/index.php/welcome.html"
  },
  "#loader": string or null, # the key you would use in the sign language datasets library. e.g. "dgs_corpus". Website will auto-link
  "#items": integer or null, # this is the number of unique signs in the column
  "#samples": string or null, # e.g. "1100 videos" or "8,257 Sentences"
  "#signers": integer or string or null, # number of unique signers
  "features": array of strings, ["feature1","feature2"], # I've seen things like "mouthing", "video:RGB", "pose:Kinect", "pose:OpenPose","text:Polish", "gloss:ASL", "writing:HamNoSys", etc.
  "language": string, # the Sign language or languages, e.g. "American" for American Sign Language (ASL)
  "license": string or null,
  "licenseUrl": string or null
}
```

Closes #32 if accepted

### CODE REVIEW

File: src/datasets/LSA-T.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "pub": {
+    "name": "LSA-T",
+    "year": 2022,
+    "publication": "dataset:dal2022lsa",
+    "url": "https://midusi.github.io/LSA-T/"
+  },
+  "features": ["video:RGB","text:es","pose:AlphaPose"],
+  "language": "Argentina",
+  "#items": null,
+  "#samples": "14,880 sentence-level videos",
```
Comment (AmitMY) [inline src/datasets/LSA-T.json:11]: 14,880 sentences

File: src/datasets/LSA-T.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "#signers": 103,
+  "license": "Public (MIT)",
```
Comment (AmitMY) [inline src/datasets/LSA-T.json:13]: just MIT

File: src/datasets/LSA-T.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "licenseUrl": "https://github.com/midusi/LSA-T/blob/d030c752e5b258f61a225612ce1767e5301ac790/LICENCE"
```
Comment (AmitMY) [inline src/datasets/LSA-T.json:14]: link should be to branch, not commit https://github.com/midusi/LSA-T/blob/main/LICENCE

File: src/references.bib [hunk 4]
```diff
@@ -3443,3 +3443,17 @@ @misc{SiMAX2020SignLanguage
...
+@inproceedings{dataset:dal2022lsa,
+ address = {Berlin, Heidelberg},
+ author = {Dal Bianco, Pedro and R\'{\i}os, Gast\'{o}n and Ronchetti, Franco and Quiroga, Facundo and Stanchi, Oscar and Hasperu\'{e}, Waldo and Rosete, Alejandro},
+ booktitle = {Advances in Artificial Intelligence – IBERAMIA 2022: 17th Ibero-American Conference on AI, Cartagena de Indias, Colombia, November 23–25, 2022, Proceedings},
+ doi = {10.1007/978-3-031-22419-5_25},
+ isbn = {978-3-031-22418-8},
+ numpages = {12},
+ pages = {293–304},
+ publisher = {Springer-Verlag},
+ title = {{LSA-T}: The First Continuous Argentinian Sign Language Dataset for Sign Language Translation},
```
Comment (AmitMY) [inline src/references.bib:3456]: {Argentinian Sign Language} 
(proper name)

**Review by AmitMY — CHANGES_REQUESTED** [review]

File: src/references.bib (hunk 4 above)
Comment (cleong110) [reply src/references.bib:3456]: Right!

File: src/datasets/LSA-T.json (hunk 3 above)
Comment (cleong110) [reply src/datasets/LSA-T.json:14]: Wouldn't a permalink be preferable though? Was trying to avoid link rot, that was what I got when I clicked "permalink" on the file in github

**Review by AmitMY — APPROVED** [review]
